src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -116,6 +116,12 @@
     <Compile Include="System\Drawing\Internal\ISystemEventTracker.cs" />
     <Compile Include="System\Drawing\Brush.cs" />
     <Compile Include="System\Drawing\Font.cs" />
+    <!-- TODO sort -->
```

Comment:
Are you going to fix this TODO in this PR or some other time?

Reply:
Ah good catch? The TODO was to actually introduced them respecting the alphabetical ordering.